AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Utils/PredictionsErrorHelper.swift

```diff
@@ -92,11 +93,38 @@ class PredictionsErrorHelper {
                 return defaultError
             }
             return AWSTranscribeStreamingErrorMessage.map(errorType) ?? defaultError
+        case NSURLErrorDomain:
+            return mapError(error)
```

Comment:
can cast to URLError now that you know it is NSURLErrorDomain, then call a specific method that takes in `URLError` as a parameter